src/products/filters/service/ProductsFiltersService.ts

```diff
@@ -0,0 +1,65 @@
+import RequestClientWrapper from '../../../RequestClientWrapper';
+import ProductsFilterRequest from '../models/ProductsFilterRequest';
+import ProductsFilter from '../models/ProductsFilter';
```

Comment:
Can you remove this extra service?

Reply:
Weird that two files duplicated when I renamed them and I completely missed this.  Removing.